modules/mapbox/src/mapbox-overlay.ts

```diff
@@ -105,7 +113,7 @@ export default class MapboxOverlay implements IControl {
     }
   }
 
-  _updateContainerSize = () => {
+  private _updateContainerSize = () => {
```

Comment:
Worth being consistent with method syntax?
```suggestion
  private _updateContainerSize() {
```